anima-jws-voucher.mkd

```diff
@@ -134,21 +127,8 @@ UTF8(STRING):
 
 # Voucher Artifact with JSON Web Signature
 
-{{RFC7515}} defines the following serializations for JWS:
```

Comment:
I agree that this introductory information had no real value and it even appears confusing.
I think it documented the exploration and decision process, which is not needed for the specification (reasoning about multiple signatures is preserved in new version).

It felt even cleaner to also remove the first subsection and now have the subsections only for the elements of the JWS structure.